examples/cloudflare-modules/src/index.ts

```diff
@@ -1,22 +1,27 @@
 // src/index.mjs
 import { createYoga, createSchema } from 'graphql-yoga'
 
-const { fetch } = createYoga({
-  graphqlEndpoint: '/graphql',
-  landingPage: false,
-  schema: createSchema({
-    typeDefs: /* GraphQL */ `
-      type Query {
-        greetings: String
-      }
-    `,
-    resolvers: {
-      Query: {
-        greetings: () =>
-          'This is the `greetings` field of the root `Query` type',
-      },
+const schema = createSchema({
+  typeDefs: /* GraphQL */ `
+    type Query {
+      greetings: String
+    }
+  `,
+  resolvers: {
+    Query: {
+      greetings: () => 'This is the `greetings` field of the root `Query` type',
     },
-  }),
+  },
 })
 
-export default { fetch }
+export default {
+  fetch(request: Request, env: { [key: string]: string }) {
+    const yoga = createYoga({
+      graphqlEndpoint: env.GRAPHQL_ROUTE || '/graphql',
```

Comment:
Also, CF module Worker doesn't use the globals for env vars, only through the `env` provided as part of `fetch`, so we need a thin wrapper in this case

Reply:
Instead of adding those extra options to our example, we can make it work by modifying the bundle like here;
https://github.com/dotansimha/graphql-yoga/pull/1812
What do you think @dotansimha @saihaj ?

Reply:
I think the tricky part of it is just that is mutates the code example